### DISCUSSION TITLE
[News] Dwie ważne nowości w blob storage

### DISCUSSION THREAD

**Discussion by kaluzaaa** [AzureCommunityPL/group#61] [News]
# [Azure Blob storage object replication is now generally available](https://azure.microsoft.com/en-us/updates/azure-blob-storage-object-replication-is-now-generally-available/)

Natywna replikacja blob pomiędzy różnymi kontami storage. Opisujemy politykami podobnie do retencji. W wielu przypadkach oznacza to koniec kodu na bazie functions i event grid do kopiowania.

- [Object replication for block blobs
](https://docs.microsoft.com/en-us/azure/storage/blobs/object-replication-overview?tabs=powershell)
- [Configure object replication for block blobs
](https://docs.microsoft.com/en-us/azure/storage/blobs/object-replication-configure?tabs=portal)

# [Change feed support in Azure Blob Storage](https://azure.microsoft.com/pl-pl/updates/change-feed-support-is-now-generally-available-for-azure-blob-storage/)

Dostępny jest podobny do cosmos db change feed w blob storage. Cytując:

> The purpose of the change feed is to provide transaction logs of all the changes that occur to the blobs and the blob metadata in your storage account. The change feed provides ordered, guaranteed, durable, immutable, read-only log of these changes

Można konsumować jako streaming albo batch.

- [Change feed support in Azure Blob Storage
](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-change-feed?tabs=azure-portal)

**Comment by mgrabarz** [comment]
Gdyby jeszcze zakodowali replikację dla Azure Storage Tables albo jakieś backupy, to już by było pozamiatane...

**Reply by ptrstpp950** [reply]
Bardzo się starają żebyś na Cosmos'a przeszedł to po co mają dodawać taki feature :)

**Reply by wi5nia** [reply]
Dokladnie jak rzecze @ptrstpp950 😉 [https://docs.microsoft.com/en-us/answers/questions/44722/is-azure-table-storage-deprecated-as-of-now.html](https://docs.microsoft.com/en-us/answers/questions/44722/is-azure-table-storage-deprecated-as-of-now.html)

**Comment by smereczynski** [comment]
Szkoda że Azure Files bez wsparcia dla event grid i change feed...

**Comment by kmadof** [comment]
A ktoś coś wie może o Last Access Time wspomnianym tutaj https://news.microsoft.com/inspire-2020-book-of-news/ ?
Jestem ciekaw kiedy coś takiego trafi fo Lifcecyle managment.

**Reply by wi5nia** [reply]
[https://azure.microsoft.com/en-us/updates/azure-blob-access-time-tracking-and-access-timebased-lifecycle-management-preview/](https://azure.microsoft.com/en-us/updates/azure-blob-access-time-tracking-and-access-timebased-lifecycle-management-preview/)

**Reply by kmadof** [reply]
Widzę że pytanie zadałem w odpowiednim momencie :)

**Reply by wi5nia** [reply]
Przypadek? Nie sadze... 😉 
Chyba nie znasz sily przelozenia niektorych czlonkow tej spolecznosci... jest tu wielu celebrytow, ktorzy jak cos chca to sie to dzieje... 😄
